src/login/login.container.js

```diff
@@ -0,0 +1,56 @@
+import React, { PropTypes } from 'react'
+import GoogleLogin from 'react-google-login'
+import { connect } from 'react-redux'
+import * as actions from './login.actions'
+import { bindActionCreators } from 'redux'
+
+export class LoginButton extends React.Component {
+
+    render () {
+        if (this.props.user.id) {
+            return (
+                <div>
+                    {this.props.user.fullname}<br/> Crédits : {this.props.user.credit}<br/>
+                    <button onClick={this.props.handleLogout}>Logout</button>
+                </div>
+            )
+        } else {
+            return (
+                <GoogleLogin
+                    clientId="673157831962-gcgp4mj9mgadau0nh9pbaikhbmqkl04d.apps.googleusercontent.com"
```

Comment:
As it turns out, we don't have any sophisticated backend yet. I ended up moving the key to a simple config constant for now.